Tests/NIOTests/EventLoopTest.swift

```diff
@@ -103,16 +103,20 @@ public class EventLoopTest : XCTestCase {
 
         let expect = expectation(description: "Is cancelling RepatedTask")
         let counter = Atomic<Int>(value: 0)
-        var repeatedTask: RepeatedTask?
-        repeatedTask = eventLoopGroup.next().scheduleRepeatedTask(initialDelay: initialDelay, delay: delay) { () -> Void in
+        let group = DispatchGroup()
+        group.enter()
+        let repeatedTask = eventLoopGroup.next().scheduleRepeatedTask(initialDelay: initialDelay, delay: delay) { () -> Void in
             if counter.load() == 0 {
                 XCTAssertTrue(DispatchTime.now().uptimeNanoseconds - nanos >= initialDelay.nanoseconds)
             } else if counter.load() == count {
                 expect.fulfill()
-                repeatedTask?.cancel()
+                group.leave()
             }
             counter.store(counter.load() + 1)
         }
+        group.notify(queue: DispatchQueue.global()) {
```

Comment:
This patch isn't acceptable, because it reductively makes this test the same as the one below. Instead, you should use `DispatchQueue.sync` to synchronise access to the repeated task variable. It's vital that this attempt to cancel from on the event loop.

Reply:
haha, thanks, didn't actually read the test name, will fix